Producer.py

```diff
@@ -0,0 +1,54 @@
+from kafka import KafkaProducer
+from json import dumps
+from time import sleep
+from pymongo import MongoClient
+from json import loads
+import random
+from flask import Flask, flash, render_template, request, jsonify
+from flask_cors import CORS, cross_origin
+from flask_pymongo import PyMongo
+import sys
+from werkzeug.utils import cached_property
+import logging
+from Event import Event
+from _datetime import datetime
+
+
+app= Flask(__name__)
+app.config['MONGO_DBNAME'] = 'offtop-kafka-mongodb'
+app.config['MONGO_URI'] = 'mongodb+srv://off-top:offtoppassword@off-top-kafka-mogsf.mongodb.net/off-top'
+app.config['MONGO_USER'] = 'off-top'
+app.config['MONGO_PASSWORD'] = 'offtoppassword'
+mongo = PyMongo()
+mongo.init_app(app)
+
+def producer():
+    producer= KafkaProducer(bootstrap_servers=['localhost:9092'],
+        value_serializer=lambda x: dumps(x).encode('utf-8'))
+    while True:
+        if(not oneZero()):
+            message= Event(random.randint(1,111), oneZero(), str(datetime.utcnow())).__dict__
+            data= dumps(message)
+            producer.send('OutgoingFocusAlert', value= data)
+            producer.flush()
+            print("Data:", data)
+            sleep(20)
+
+def oneZero():
```

Comment:
I'd recommend improving readability by changing this method to something like getRandomFocusScore()